pkgs/by-name/br/brlaser/package.nix

```diff
@@ -18,6 +19,14 @@ stdenv.mkDerivation rec {
     hash = "sha256-a+TjLmjqBz0b7v6kW1uxh4BGzrYOQ8aMdVo4orZeMT4=";
   };
 
+  patches = [
```

Comment:
Please add a comment referencing the release number in which the fix is expected to appear